site/book/06-deploying-packages/03-handling-dependencies.md

```diff
@@ -0,0 +1,74 @@
+Sometimes resources within a package have dependencies that require
+one resource to be applied and reconciled before the next resource
+is applied. For example, a package that includes both Wordpress and
+MySQL might require that MySQL is running before Wordpress is started.
```

Comment:
You may want quality you mean by `Wordpress` and `Mysql` (i.e. a deployment not a package)